drivers/hmis/lib/ce_workflows/ac_builder.rb

```diff
@@ -0,0 +1,381 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+##
+# CeWorkflows::AcBuilder
+# Utility for building Coordinated Entry (CE) workflow definitions specific to the AC implementation.
+# Intended for use in scripts and Rake tasks that automate the setup or teardown of AC CE workflows.
+# Not intended for use in production application logic.
+module CeWorkflows
+  class AcBuilder
+    CE_STEP_FORMS = {
+      initial_review: 'housing_workflow_initial_review',
+      initial_client_engagement: 'housing_workflow_initial_client_engagement',
+      client_engagement: 'housing_workflow_client_engagement',
+      client_offer_outcome: 'housing_workflow_client_offer_outcome',
+      provider_outcome_1: 'housing_workflow_provider_outcome_1',
+      provider_outcome_2: 'housing_workflow_provider_outcome_2',
+      provider_outcome_3: 'housing_workflow_provider_outcome_3',
+      denial_review_1: 'housing_workflow_denial_review_1',
+      denial_review_2: 'housing_workflow_denial_review_2',
+      denial_review_3: 'housing_workflow_denial_review_3',
+      confirm_success: 'housing_workflow_confirm_success',
+      initial_outgoing_referral: 'admin_assign_workflow_initial_outgoing_referral',
+    }.freeze
+
+    def initialize(data_source)
+      @data_source = data_source
+
+      # validate forms exist
+      missing_identifiers = CE_STEP_FORMS.values - Hmis::Form::Definition.where(role: 'CE_REFERRAL_STEP', identifier: CE_STEP_FORMS.values).pluck(:identifier)
+      raise "Some form definitions are missing. Did you run 'rails driver:hmis:seed_definitions'? #{missing_identifiers.join(', ')}" if missing_identifiers.any?
+    end
+
+    # This method builds the QA housing workflow version 1, which is a referral workflow for housing opportunities
```

Comment:
This comment has me a bit confused, I think it's the "QA" part. We are working towards this being the prod AC workflow, right?
```suggestion
    # This method builds the AC housing workflow version 1, which is a referral workflow for housing opportunities
```